src/.defaultsForValidator.js

```diff
@@ -65,7 +65,7 @@ const defaults = {
       'no_property_description': 'warning',
       'description_mentions_json': 'warning',
       'array_of_arrays': 'warning',
-      'inconsistent_property_type': 'warning',
+      'inconsistent_property_type': ['warning', 'code', 'default', 'type', 'value'],
```

Comment:
Would it make more sense this way?
```suggestion
      'inconsistent_property_type': ['warning', ['code', 'default', 'type', 'value']],
```